### PR TITLE
[WOR-1416] Update gradle, jib, and two other libraries required by JShell.

### PR DESCRIPTION
**Pull request by cahrens**
Originally https://github.com/DataBiosphere/terra-workspace-manager/pull/1560.

### CODE REVIEW

**Comment by cahrens** [comment]
This is currently failing because of the validation added in https://github.com/GoogleContainerTools/jib/pull/3550

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.3-bin.zip
```
Comment (cahrens) [inline gradle/wrapper/gradle-wrapper.properties:3]: The jib update required upgrading Gradle.

File: build.gradle [hunk 2]
```diff
@@ -4,8 +4,8 @@ plugins {
 
     // For JShell support
     // https://gitlab.com/barfuin/gradle-taskinfo
-    id 'org.barfuin.gradle.taskinfo' version '1.1.1'
-    id "com.github.mrsarm.jshell.plugin" version "1.1.0"
+    id 'org.barfuin.gradle.taskinfo' version '2.1.0'
```
Comment (cahrens) [inline build.gradle:7]: Upgrading Gradle required upgrading gradle-taskinfo. I decided to also update jshell.plugin at the same time, though it was not technically required.

File: azureDatabaseUtils/gradle/deploy.gradle [hunk 3]
```diff
@@ -1,12 +1,11 @@
-import java.time.ZonedDateTime
-
 // Deploy config
 jib {
   from {
     // docker:// specifies a local image
     image = "docker://azure-database-utils-base:latest"
   }
   extraDirectories {
+    mkdir jibExtraDirectory
```
Comment (cahrens) [inline azureDatabaseUtils/gradle/deploy.gradle:8]: A contribution to jib added validation of the directory specified here. It fails if the directory does not yet exist.

File: build.gradle (hunk 2 above)
Comment (cahrens) [reply build.gradle:7]: To be honest though, I have no idea how to tell if JShell still works with these changes.

File: azureDatabaseUtils/gradle/deploy.gradle [hunk 4]
```diff
@@ -1,12 +1,11 @@
-import java.time.ZonedDateTime
```
Comment (cahrens) [inline azureDatabaseUtils/gradle/deploy.gradle:1]: Was unused.

File: build.gradle (hunk 2 above)
Comment (sergiygetlin) [reply build.gradle:7]: I wonder how/why JShell was used here? I don't see that 'jshell' task is used anywhere. Maybe we don't need it at all...

**Review by blakery — APPROVED** [review]

**Review by sergiygetlin — APPROVED** [review]
LGTM; maybe we need to revise usage of jshell later. Quick search doesn't show any usage of jshell. At least I haven't found any usage of jshell task.

File: build.gradle (hunk 2 above)
Comment (cahrens) [reply build.gradle:7]: Sorry, I didn't notice the comment before I hit merge.

@ddietterich add the JShell support. Dan, is it something that you use?

**Pull request merged**